wordpress-mobile/AztecEditor-Android
# Simplify containsPreformat method

### Fix
The `containsPreformat` method doesn't work well when the preformat doesn't contain any code. In this PR I'm simplifying it so that it works in all cases. I don't know why it was originally so complicated. I think the original behaviour doesn't have any benefits.

### Test
Enable preformat action by adding it to toolbar actions
```
        val defaultBasicLayout = BasicLayout(
                ToolbarAction.PREFORMAT,
...
```
to make it easier set the `EXAMPLE` in the `MainActivity` to something like this: 
```
"""
<pre></pre>
<p>paragraph 1</p>
""".trimIndent()
```

1. Run the app
2. Click into the preformat line
3. Trigger the toolbar preformat action
4. Notice the `preformat` disappears and is not duplicated

### Review
@khaykov 

Make sure strings will be translated:
